### PR TITLE
fix: add reviewers in for dependabot config

### PR DESCRIPTION
**Pull request by zubairshakoorarbisoft**
We will be using the following command to pass thee reviewers to add them in ecosystems reviewers:
`add_dependabot_ecosystem --reviewer "edx/arbi-bom"`

reviewer is tha combination of org name/team name, for more details have a look into its [docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#reviewers)

### CODE REVIEW

File: edx_repo_tools/dependabot_yml.py [hunk 1]
```diff
@@ -36,8 +41,22 @@ def _add_elements(self):
         if not found:
             self.elements['updates'].append(self.yml_instance.load(value))
 
+
```
Comment (iamsobanjaved) [inline edx_repo_tools/dependabot_yml.py:44]: can you remove an extra line break from here, https://peps.python.org/pep-0008/#blank-lines

File: edx_repo_tools/dependabot_yml.py [hunk 2]
```diff
@@ -36,8 +41,22 @@ def _add_elements(self):
...
+        if self.reviewer:   
+            self.elements['updates'] = self.elements.get('updates') or []
+            for key, value in ADD_NEW_FIELDS:
+                for index, elem in enumerate(self.elements['updates']):
+                    if key == elem.get('package-ecosystem'):
+                        self.elements["updates"][index].update(self.yml_instance.load(
+                            ecosystem_reviewers.format(**{"reviewer": self.reviewer})
+                        ))
+                        break
+
```
Comment (iamsobanjaved) [inline edx_repo_tools/dependabot_yml.py:55]: Similar, need to remove extra line gap

File: edx_repo_tools/dependabot_yml.py [hunk 3]
```diff
@@ -36,8 +41,22 @@ def _add_elements(self):
         if not found:
             self.elements['updates'].append(self.yml_instance.load(value))
 
+
+    def _add_reviewers(self):
+        if self.reviewer:
```
Comment (iamsobanjaved) [inline edx_repo_tools/dependabot_yml.py:46]: Can you move this if condition to the call of this function to reduce extra indentation in this function?

File: edx_repo_tools/dependabot_yml.py [hunk 4]
```diff
@@ -48,15 +67,18 @@ def modernize(self):
 @click.option(
     '--path', default='.github/dependabot.yml',
     help="Path to target dependabot.yml file")
-def main(path):
+@click.option(
+    '--reviewer', default=None,
+    help="Name of the reviewer")
```
Comment (iamsobanjaved) [inline edx_repo_tools/dependabot_yml.py:72]: Kindly add the info that the reviewer should be a github username or team name, and the team name should be in the format of `org-name/team-name`.

**Review by iamsobanjaved — COMMENTED** [review]
Just few suggestions

File: edx_repo_tools/dependabot_yml.py (hunk 4 above)
Comment (zubairshakoorarbisoft) [reply edx_repo_tools/dependabot_yml.py:72]: sure

File: edx_repo_tools/dependabot_yml.py (hunk 3 above)
Comment (zubairshakoorarbisoft) [reply edx_repo_tools/dependabot_yml.py:46]: sure

File: edx_repo_tools/dependabot_yml.py (hunk 2 above)
Comment (zubairshakoorarbisoft) [reply edx_repo_tools/dependabot_yml.py:55]: done

File: edx_repo_tools/dependabot_yml.py (hunk 1 above)
Comment (zubairshakoorarbisoft) [reply edx_repo_tools/dependabot_yml.py:44]: done

**Review by iamsobanjaved — APPROVED** [review]

**Pull request merged**